Index: totem/src/meson.build
===================================================================
--- totem.orig/src/meson.build
+++ totem/src/meson.build
@@ -295,14 +295,14 @@ libtotem_gir = gnome.generate_gir(
   extra_args: gir_extra_args
 )
 
-tests = [ 'test-totem' ]
-
-foreach test_name : tests
-  exe = executable(test_name, '@0@.c'.format(test_name),
-                   include_directories: [src_inc, top_inc],
-                   dependencies: libtotem_dep)
-
-  test(test_name, exe)
-endforeach
+#tests = [ 'test-totem' ]
+#
+#foreach test_name : tests
+#  exe = executable(test_name, '@0@.c'.format(test_name),
+#                   include_directories: [src_inc, top_inc],
+#                   dependencies: libtotem_dep)
+#
+#  test(test_name, exe)
+#endforeach
 
 subdir('plugins')
